src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,20 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
```

Comment:
block cant have external indents

[en.bem.info/methodology/css/#external-geometry-and-positioning](https://en.bem.info/methodology/css/#external-geometry-and-positioning)